diff --git a/bot/__main__.py b/bot/__main__.py
index bf37d7c..2e9c9e2 100644
--- a/bot/__main__.py
+++ b/bot/__main__.py
@@ -31,7 +31,6 @@ class MatrixBot:
                 msg for msg in response.chunk
                 if isinstance(msg, RoomMessageImage)
                 and "url" in msg.source["content"]  # only media messages have 'url'
-                and int(MINIMAL_AGE) <= int(msg.source["age"])
                 ])

             if token is None:  # all messages have been fetched
